### PR TITLE
T511-017: make stylechecks work with pycodestyle

### PR DESCRIPTION
**Pull request by raph-amiard**

### CODE REVIEW

File: langkit/lexer/__init__.py [hunk 1]
```diff
@@ -879,7 +879,7 @@ def __init__(self, match_length, *alts):
             self.location = extract_library_location()
             self.match_length = match_length
 
-            for i, alt in enumerate(alts):
+            for _, alt in enumerate(alts):
```
Comment (pmderodat) [inline langkit/lexer/__init__.py:882]: Let’s remove the call to enumerate altogether

File: langkit/stylechecks/__init__.py [hunk 2]
```diff
@@ -452,25 +452,36 @@ def check(self, report, filename, content, parse):
         self.custom_check(report, filename, content, parse)
         if os.path.exists(filename):
             self.pep8_check(report, filename)
-            self.pyflakes_check(report, filename, content)
+            # self.pyflakes_check(report, filename, content)
```
Comment (pmderodat) [inline langkit/stylechecks/__init__.py:455]: Is this permanent? Can you explain?

File: langkit/lexer/__init__.py (hunk 1 above)
Comment (raph-amiard) [reply langkit/lexer/__init__.py:882]: Right ... :derp:

File: langkit/stylechecks/__init__.py (hunk 2 above)
Comment (raph-amiard) [reply langkit/stylechecks/__init__.py:455]: It's a mistake ... Or the reason code reviews exists :) Thanks !

**Pull request merged**